services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -5331,7 +5331,15 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/put/responses/200/content/application~1json/schema'
+                type: object
```

Comment:
TIP: add in the PR message a link to see the new OAS using the raw link to ``openapi.yaml`` as 

https://editor.swagger.io/?url=https://raw.githubusercontent.com/ITISFoundation/osparc-simcore/927c7480febc2cfd915efc2b99678873425ff491/services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

Reply:
cool tip, it was added in the PR comment section :+1: